test/utils/run_tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+PATH=$PATH:node_modules/.bin
+
+runTest ( ) {
+    if [ "$1" = "sqlite" ]
+    then
+        echo "Running with SQLite backend"
+        export RB_TEST_BACKEND=sqlite
+        rm -f restbase
+    else
+        echo "Running with Cassandra backend"
+        export RB_TEST_BACKEND=cassandra
+        sh ./test/utils/cleandb.sh
+    fi
+
+    if [ "$2" = "test" ]
```

Comment:
Better to compact these to `if` blocks into `if`-`elsif` - `else` so that we can be sure a good command has been given (it's also more readable)